analyze.py

```diff
@@ -346,11 +346,13 @@ def generate_test_config(analysis_mode, project, log_dir, two_sided, causality_g
 def analyze_trace(
     project,
     log_dir,
+    data_dir,
     generate_oracle=True,
     generate_config=True,
     two_sided=False,
     use_sql=False,
     compress_trivial_reconcile=True,
+    learn_twice=False,
```

Comment:
Can you change all the bool `learn_twice` to sth like `canonicalize_resource` for disambiguation?